tests/components/vizio/test_media_player.py

```diff
@@ -746,6 +746,8 @@ async def test_apps_update(
                 "homeassistant.components.vizio.gen_apps_list_from_url",
                 return_value=APP_LIST,
             ):
+                async_fire_time_changed(hass, dt_util.now() + timedelta(days=2))
+                await hass.async_block_till_done()
                 async_fire_time_changed(hass, dt_util.now() + timedelta(days=2))
```

Comment:
This integration has two ways to update, from a coordinator, and from polling. The test did block for both updates and previously relied on task being created in entity_platform slowing things down though that it got another event loop run.